bin/chef-cluster

```diff
@@ -0,0 +1,115 @@
+#!/bin/sh
+#
+# Copyright 2017, SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+cmd=$1
+workercount=${2:-auto}
+vhostconfig=/etc/apache2/vhosts.d/chef-cluster.conf
+serverconfig=/etc/sysconfig/chef-server
+webroot=/srv/www/chef-cluster
+
+baseport=4001
+
+balancerstart=BEGIN-BALANCER-MEMBER
+balancerend=END-BALANCER-MEMBER
+
+usage() {
+  echo "usage: chef-cluster <enable|disable> [<workercount>|auto]"
+  exit 1
+}
+
+case $cmd in
+  enable)
+    [ "$workercount" == "auto" ] && workercount=$(grep -c ^processor /proc/cpuinfo)
+    [ "$workercount" -ge 2 ] || { echo "ERROR: workercount must be >= 2"; exit 2; }
+
+    # apache configuration
+    mkdir -p "$webroot"
+    a2enmod -q lbmethod_byrequests || a2enmod lbmethod_byrequests
+    test -e "$vhostconfig" || cat <<EOF > $vhostconfig
```

Comment:
Don't we want to always write the file, in case we fix the template in the script?

Reply:
1. if you disable the balancer and enable it again you will get a new file
2. I was thinking more about a case where someone would modify some of the file (e.g. log paths) and would like to change number of workers later. With current implementation the changes won't be lost.

Reply:
One nice usecase where user might want to modify the generated vhost file is opening access to `balancer-manager` status page from outside of localhost.